volume/drivers/aws/aws_storage.go

```diff
@@ -226,6 +228,18 @@ func (s *ec2Ops) ApplyTags(
 	return err
 }
 
+func (s *ec2Ops) RemoveTags(
+	v *ec2.Volume,
+	labels map[string]string,
+) error {
+	req := &ec2.DeleteTagsInput{
+		Resources: []*string{v.VolumeId},
```

Comment:
That's weird that the SDK requires a `*string`. Aren't strings already a reference? Still code looks good.

Reply:
i think the only time you want to pass string pointers is to differentiate between empty string and no string (i.e. nil). Another use case would be to set those values - since golang strings are immutable - so you need to pass its pointer to modify its value.